code/modules/trade/datums/trade_stations_presets/1-common/ghost_kitchen.dm

```diff
@@ -276,6 +276,7 @@
 		/obj/item/reagent_containers/glass/rag = offer_data("Rag", 40, 5),
 		/obj/item/reagent_containers/spray = offer_data("Spray Bottle", 40, 10),
 		/obj/item/mop = offer_data("Mop", 150, 2),
-		/obj/item/reagent_containers/food/snacks/grown = offer_data("spare grown food", 10, 120)
+		/obj/item/reagent_containers/food/snacks/grown = offer_data("spare grown food", 10, 120),
+		/obj/item/tray = offer_data("dinner tray", 125, 5)
```

Comment:
Same thing as the box, this can hold items thus destory items that are not meant to be